starter_templates/cpp/code/.codecrafters/run.sh

```
@@ -8,4 +8,4 @@
 
 set -e # Exit on failure
 
-exec $(dirname "$0")/build/git "$@"
+exec $(dirname $0)/build/git "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)